robots_101/programme_structure.md

```diff
@@ -69,8 +69,8 @@ as volunteers able to help with the kits and hands-on guidance with your robot.
 
 ## Virtual League
 
-The Virtual League is often the first change you'll have to test your
-strategies against other teams. You'll submit your robot code for running
+The Virtual League is often the first chance you'll have to test your strategies
+against other teams. You'll [submit][code-submitter] your robot code for running
 against other teams in the [simulator][simulator]. The matches are livestreamed
 on our [YouTube Channel][youtube] and you'll get a copy of your robot's logs
```

Comment:
> and you'll get a copy of your robot's logs

Pedantry: Don't they also get their match recording as well?

Reply:
Yeah, probably. Feel free to open a PR for that. I'd rather keep this one focussed on the code submitter.